lib/tasks/foreman_plugin_template_tasks.rake

```diff
@@ -13,11 +13,10 @@ end
 # Tests
 namespace :test do
   desc 'Test ForemanPluginTemplate'
-  Rake::TestTask.new(:foreman_plugin_template) do |t|
-    test_dir = File.expand_path('../../test', __dir__)
-    t.libs << 'test'
-    t.libs << test_dir
-    t.pattern = "#{test_dir}/**/*_test.rb"
+  Rake::TestTask.new(:foreman_plugin_template => 'db:test:prepare') do |t|
+    t.libs << ForemanPluginTemplate::Engine.root.join('test')
```

Comment:
Before this, `libs` was `['test', File.expand_path('../../test', __dir__)]`, now it's only ` [ForemanPluginTemplate::Engine.root.join('test')]`, doesn't that mean the bare `test` one is not added anymore, which means the test helpers from Foreman are not available anymor?

Reply:
We just tried that and this is exactly what happened: https://github.com/ATIX-AG/foreman_resource_quota/actions/runs/12009329940/job/33473856106#step:21:18